bionty/core/_bionty.py

```diff
@@ -17,18 +18,24 @@ def sync_all_sources_to_latest():
     """
     import lamindb as ln
 
+    import bionty
```

Comment:
You are importing all of bionty but then also importing specific things from bionty below. I guess this can be consolidated. It's not a big deal.

(I see how you use it here `list_biorecord_models(bionty)`

Reply:
yes, it has to access to the entire module because it needs to know which modules are there. i'll figure out something better in the futrue :)